lib/constants.ts

```diff
@@ -30,6 +31,12 @@ export enum RoutingType {
   DUTCH_V2 = 'DUTCH_V2',
 }
 
+export enum QuoteType {
+  CLASSIC,
+  RFQ,
+  SYNTHETIC,
```

Comment:
I think it's time to make the move to "OPEN" in our terminology instead of synthetic

Reply:
<img width="205" alt="image" src="https://github.com/Uniswap/unified-routing-api/assets/11261382/1d87ab1b-9a15-4479-a148-cf0c48149948">
Oh boy.. how about in a new PR?

Reply:
Oh jeez LOL yea def